src/components/TodoListScreen.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+  Button,
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+
+class TodoListScreen extends Component {
+  render(){
+    const { navigation, allTodos} = this.props;
+    const todosList = allTodos.map((todo, index) => 
+      <Text key={index} style={styles.todoItem}>{todo}</Text>);
+    return (
+      <View style={styles.container}>
```

Comment:
O que acontece se adicionares 100 Todos? Em mobile tens o componente **ScrollView** que te parmite fazer scroll

```suggestion
      <ScrollView style={styles.container}>
```